src/sqlfluff/rules/L061.py

```diff
@@ -0,0 +1,54 @@
+"""Implementation of Rule L061."""
+
+from typing import Optional
+
+from sqlfluff.core.parser.segments.raw import CodeSegment
+from sqlfluff.core.rules.base import BaseRule, LintFix, LintResult, RuleContext
+from sqlfluff.core.rules.doc_decorators import document_fix_compatible
+
+
+@document_fix_compatible
+class Rule_L061(BaseRule):
+    """Use ``!=`` instead of ``<>`` for ``not equal to`` comparison.
+
+    | **Anti-pattern**
+    | ``<>`` means ``not equal`` but doesn't sound like this when
+    | we say it loud.
+
+    .. code-block:: sql
+
+        SELECT * FROM X WHERE 1 <> 2;
+
+    | **Best practice**
+    | Use ``!=`` instead because it's sounds more natural and
+    | is more common in other programming languages.
+
+    .. code-block:: sql
+
+        SELECT * FROM X WHERE 1 != 2;
+
+    """
+
+    def _eval(self, context: RuleContext) -> Optional[LintResult]:
+        """Use ``!=`` instead of ``<>`` for ``not equal to`` comparison."""
+        # Only care about not_equal_to segments.
+        if context.segment.name != "not_equal_to":
+            return None
+
+        # Only care about ``<>``
+        if "<>" not in context.segment.raw:
+            return None
+
+        # Provide a fix and replace ``<>`` with ``!=``
+        fix = LintFix.replace(
+            context.segment,
+            [
+                CodeSegment(
+                    raw="!=",
+                    name="not_equal_to",
+                    type="comparison_operator",
+                )
+            ],
+        )
```

Comment:
Again given my explanation above you shouldn't be fixing the "not_equal_to" segment but it's raw children.

so maybe something like:
```python
fixes = [
    LintFix.replace(
        raw_comparison_operators[0],
        [
            CodeSegment(
                raw="!",
                name="raw_not"
                type="raw_comparison_operator"
            )
        ],
    ),
    LintFix.replace(
        raw_comparison_operators[1],
        [
            CodeSegment(
                raw="=",
                name="raw_equals"
                type="raw_comparison_operator"
            )
        ],
    ),
]
```
Otherwise we'd end up with an incorrect parse tree

Reply:
This I do agree with however. We should make this change.